game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/HelpTexts.java

```diff
@@ -0,0 +1,135 @@
+          + "Copy the topic id from the URL displayed in the address bar of you "
+          + "browser. If the URL is "
+          + "http://www.axisandallies.org/forums/index.php?topic=25252.0, "
+          + "put this 5 digit number (25252) into the Topic ID field in TripleA<br/>"
+          + "Put your username and password for the AxisAndAllies.org forum into the "
+          + "username and password fields<br/>"
+          + "Click the Test Post button, to check that TripleA can reply to your forum "
+          + "post<br/>"
+          + "<i>Note:</i> Your AxisAndAllies.org username and password are not stored as part "
+          + "of the save game, but they are stored encrypted in the local file system "
+          + "if you select the option to remember your credentials. You may have to "
+          + "enter your username and password again if you open the save game on "
+          + "another computer."
+          + "</p></html>";
+  public static final String SMTP_DISABLED =
+      "<html><p style='width: 400px;'>"
+          + "Email sender<br/>"
+          + "An email sender can email the turn summary and save game to multiple "
+          + "recipients at the end of each players turn. This allows two or more "
+          + "players to play a game where you don't have to be online at the same "
+          + "time.<br/>"
+          + "Each email sender may require custom configuration, to learn more "
+          + "click help again after selecting a specific email sender."
+          + "</p></html>";
+
+  public static final String GENERIC_SMTP_SERVER =
+      "<html><p style='width: 400px;'>"
+          + "Email through SMTP<br/>"
+          + "This email sends email via any generic SMTP service. Configuration:<br/>"
+          + "<b>Subject:</b> This will be the subject of the email. In addition to the text "
+          + "entered, the player and round number will be appended<br/>"
+          + "<b>To:</b> A list of email addresses separated by space. the email will be sent "
+          + "to all these users<br/>"
+          + "<b>Login:</b>: Your login to the smtp server<br/>"
+          + "<b>Password:</b> Your password smtp server<br/>"
+          + "<b>Host:</b> The host address (or ip) of the SMTP server<br/>"
+          + "<b>Post:</b> The post of the smtp server (typically 25 for unencrypted, and 587 "
+          + "for TLS encrypted servers)<br/>"
+          + "Use TLS encryption: check this if yor server supports the TLS protocol for "
+          + "email encryption<br/>"
+          + "<i>Note:</i> Your SMTP login and password are not stored as part of the save game, "
+          + "but they are stored encrypted in the local file system if you select the "
+          + "option to remember your credentials.<br/>"
+          + "You may have to enter your login and password again if you open the save "
+          + "game on another computer."
+          + "</p></html>";
```

Comment:
Also thoughts about moving this to a resource html file?
HTML in source code is alway kind of weird.

Reply:
IMO it's overkill. A class with dedicated constants and lots of HTML IMO is fine, avoids complexity of making sure a resource file is in the right place. Eventually this is going to be behind a string identifier and presumably translated to multiple languages, I'd be concerned about code churn for anything short of that update.